### DISCUSSION TITLE
relation between piper and espeak

### DISCUSSION THREAD

**Discussion by siddjain** [rhasspy/piper#582] [Q&A]
I noticed that piper depends on [espeak](https://github.com/espeak-ng/espeak-ng/) which itself is a full-fledged TTS engine if I am not mistaken. So what value-add does piper provide on top of espeak? Is piper simply a wrapper? can someone help me understand?

PS: Also if espeak uses GPL License how can Piper use MIT License?

**Comment by synesthesiam** [comment]
Previous discussion here: https://github.com/rhasspy/piper/issues/93

In short:
* espeak is used to get phonemes only. Piper uses a neural network model vs. espeak's basic wave synthesis. Listen to espeak vs. piper and you will hear what it adds 😄
* espeak is technically optional, and will be replaced entirely in the next version of piper to avoid any possible licensing issues

**Reply by KleberNoel** [reply]
When will the new version without espeak be available? :D

**Reply by Please-just-dont** [reply]
I too am wondering this. Will the replacement for espeak be written also in C++? And will it will be buildable as a library instead of an executable?

**Comment by luiscarlos2000** [comment]
I was thinking the same. What will happen when this version be released? Will any new phonemicers be added besides eSpeakNG's?
